src/components/CounterTabs/CounterTabs.js

```diff
@@ -2,17 +2,17 @@ import React from 'react';
 import CounterIcon from '../CounterIcon';
 
 
-const CounterTabs = ({tabClick, heroID, currentTab}) => {
-    console.log(`HERO: ${heroID}\n CURRENT" ${currentTab}\n`);
+const CounterTabs = ({tabClick, heroID, currentTab, counter}) => {
+
     return (
             <div onClick={() => tabClick(heroID)} className="tab col-1-of-3">
                     <div className={currentTab === heroID ? "row tab-border tab-active" : 
                     "row tab-border"}>
-                        <div className="col-1-of-2 tab-height tab-padding">
-                            <CounterIcon size="150" />
-                        </div>
+                    <div className="col-1-of-2 tab-height tab-padding">
+                    {counter.icon_url && <CounterIcon counterIcon={counter.icon_url} />}
+                    </div>
                         <div className="col-1-of-2 tab-height tab-padding tab-relative">
-                            <h3 className="vt-ht-center">Hanzo</h3>
+                            <h3 className="vt-ht-center">{counter && counter.name}</h3>
```

Comment:
I understand what you're doing with those `&&` here, but consider both the empty `<h3>` in case there's no `counter` and if that's something you're "accepting" as a consequence or deliberately "doing" because it's what you want to happen.

Also, consider that && is a boolean operator, and while your code will work here, you're not actually using a boolean on the left, which someone might expect, then imagine their surprise when you access a property on what appeared to be a boolean value.

Reply:
Here I'm counting on Javascript's idea of "Falsy" and "Truthy". Honestly this operation was just for practice. I enjoy the ternary operator more. Should I be keeping a default in case a name was not input in the data?

Reply:
The road to heaven is paved with lines of code we write to protect our users from invalid input.